### ISSUE TITLE
Bold font doesn't work in Mac version of Marp 0.0.10

### ISSUE THREAD

**Issue by soramame0518** [yhatt/marp#163]
If I format texts using bold (e.g., **test**) , the Marp's viewer (right window) shows it as bold, but the exported PDF doesn't. This might be a bug in Mac version, as I confirmed that Windows version works properly.

**Comment by daturkel** [comment]
Having this problem as well.

**Comment by paulrolfe** [comment]
Yeah, this is pretty annoying. It also won't render emojis in the PDF.

I think these PDF export bugs would be OK, if Marp had a presentation mode. (Feature Request)
